### ISSUE TITLE
Add the ability to update config file from within the web UI?

### ISSUE THREAD

**Issue by whitedavidp** [thess/callattendant#50]
I have been so happy with my own CA installation, I have convinced some friends - who are also inundated with spam calls - to try it out on an old Android TV box that I have converted to run Armbian. Initial setup shows it is working and that is impressive!

However, these users are not very tech-saavy and the CA machine will run headless, connected by Wifi so they can get to the UI from their own web browsers. For 90% of things, I think that will be ok. And, in a pinch, I should be able to reach the box via ssh from one of their computers while I remote control it using Team Viewer for the remaining 10% - like clearing the call log in the database periodically and making config changes.

But most users might not have this capacity. I have already written on the subject of clearing the call log from the user interface... But I cannot help but wonder if there isn't some fairly straight forward way to change the config file from the user interface and restart CA? It is great that we can see/study the config file via the UI. But it would be great if the Settings page offered a button to edit the file's contents, save (after creating a backup), and re-start CA.

I fully realize that this is a loaded gun. And there should be warnings a-plenty. But it might bring at least simple changes into the reach of less technical users.

Thanks as always!

**Comment by thess** [comment]
I agree, it would be nice -- If someone else doesn't submit a PR with this feature, I might do it some day.
